pkg/containers/detection.go

```diff
@@ -0,0 +1,33 @@
+package containers
+
+import "os"
+
+// IsRunningInContainer checks whether the current process is being executed inside one of the containers we check for
+func IsRunningInContainer() bool {
```

Comment:
how about adding a test for it?
when on github it should return true otherwise skip as someone may run it locally